src/model/FaceliftConversion.h

```diff
@@ -470,7 +470,7 @@ struct TypeHandler<QList<ElementType> >
         Q_UNUSED(receiver);
         Q_UNUSED(function);
         Q_UNUSED(connections);
-        static_assert(sizeof(ElementType) >= 0, "Missing specialization");
+        static_assert(sizeof(ElementType) < 0, "Missing specialization");
```

Comment:
This looks suspicious. Could you explain this change?

Reply:
This fixes a mistake. sizeof() always return a positive value so the assertion was always true, which means it was useless.